images/linux/scripts/installers/1604/hosted-tool-cache.sh

```diff
@@ -31,14 +31,23 @@ DocumentInstalledItem "Python:"
 pythons=$(ls $AGENT_TOOLSDIRECTORY/Python)
 for python in $pythons; do
 	DocumentInstalledItemIndent "Python $python"
+	# Add symlinks for python, python2 and python3 so that Python can be used out of the box without setup-python
+	if [ ${python:0:3} == "2.7" ] ; then
+		# add python 2.7 for python and python2
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python
+		ln -s $AGENT_TOOLSDIRECTORY/Python/$python/x64/bin/python /usr/local/bin/python2
+	elif [ ${python:0:3} == "3.5" ] ; then
+		# add python 3.5 for python3 which is the default for 1604
```

Comment:
For both this and Ubuntu 18, we can take the opportunity to update to the latest Python.  The only reason we were using an older version was because of what was available through APT.